src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +23,23 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        Optional<User> optionalUser = Optional.ofNullable(user);
+        return optionalUser
+                .filter(o -> o.getAge() != null)
```

Comment:
```suggestion
                .filter(User::isAgeNotNull)
```

- 의미를 드러내는 이름과함께 메소드로 분리시켜도 좋을것 같네요 😄 
- `get-`를 지양하기 위해 객체에게 물어보어요~